packages/core/src/extension/extension-message.ts

```diff
@@ -0,0 +1,16 @@
+import { Consignment } from '../shipping';
+
+import { ExtensionEvent } from './extension-events';
+
+export const enum ExtensionMessageType {
+    CurrentConsignments = 'EXTENSION:CURRENT_CONSIGNMENTS',
```

Comment:
We can just call it `GetConsignments` instead of CurrentConsignments 🤔

Reply:
Updated.